src/main/java/org/netway/dongnehankki/post/domain/Comment.java

```diff
@@ -30,4 +32,18 @@ public class Comment extends BaseEntity {
 	@ManyToOne(fetch = FetchType.LAZY)
 	@JoinColumn(name = "user_id", nullable = false)
 	private User user;
+
+	public static Comment createComment(String content, Post post, User user) {
+		return new Comment(content, post, user);
+	}
+
+	private Comment(String content, Post post, User user) {
+		this.content = content;
+		this.post = post;
+		this.user = user;
+	}
+
+	public void update(String content) {
```

Comment:
```suggestion
    public void updateComment(String content) {
```
좀 더 명확하게 이건 어떤가....해서

Reply:
수정 완료~